resident/resident-service/pom.xml

```diff
@@ -147,7 +147,7 @@
 		<dependency>
```

Comment:
Always have the jira Id with the exact format in commit messages and it the PR titile

Reply:
yes added.

Reply:
ok resolved